fili-core/src/main/java/com/yahoo/bard/webservice/web/responseprocessors/FullResponseProcessor.java

```diff
@@ -39,4 +47,21 @@
  *
  */
 public interface FullResponseProcessor extends ResponseProcessor {
+    @Override
+    default void processResponse(JsonNode json, DruidAggregationQuery<?> query, LoggingContext metadata) {
```

Comment:
Are you sure? I think just overriding the method as an interface without default should be fine. Let me know the compilation error you get if possible.